assets/javascripts/views/layout/document.coffee

```diff
@@ -46,6 +46,27 @@ class app.views.Document extends app.View
     app.appCache?.updateInBackground()
     return
 
+  hideSidebar: (saveLayout = true) ->
```

Comment:
Rather than having extra `hideSidebar` / `showSidebar` functions, I'd prefer we use and extend the existing `toggleSidebar` function (maybe with a boolean argument).